src/input.rs

```diff
@@ -377,7 +412,7 @@ impl<'a> Input<'a> {
     }
 
     fn set_cursor(&mut self) -> io::Result<()> {
-        // if we have a placeholer, move the cursor left to beginning of the input
+        // if we have a placeholder, move the cursor left to beginning of the input
         if !self.placeholder.is_empty() && self.input.is_empty() {
```

Comment:
it seems this does not work anymore after the changes and the cursor is placed after the placeholder.

<img width="516" alt="Screenshot 2024-04-26 at 09 51 46" src="https://github.com/jdx/demand/assets/181773/b918ec3e-e591-49a2-9de6-0604f9ce4105">

as opposed to the beginning

<img width="528" alt="Screenshot 2024-04-26 at 09 54 41" src="https://github.com/jdx/demand/assets/181773/12f6a580-de3a-403d-997c-aac6e7cc57a8">